src/main/java/rx/internal/util/unsafe/MessagePassingQueue.java

```diff
@@ -29,7 +29,7 @@
  * 
  * @param <M> the event/message type
  */
-interface MessagePassingQueue<M> {
+public interface MessagePassingQueue<M> {
```

Comment:
Ah, because spsc queues are public but interface they're implementing is
not. Ok then!

On Mon, 20 Jun 2016, 16:00 David Karnok, <notifications@github.com> wrote:

> In src/main/java/rx/internal/util/unsafe/MessagePassingQueue.java
> <https://github.com/ReactiveX/RxJava/pull/4039#discussion_r67684701>:
>
> > @@ -29,7 +29,7 @@
> >   *
> >   * @param <M> the event/message type
> >   */
> > -interface MessagePassingQueue<M> {
> > +public interface MessagePassingQueue<M> {
>
> javadoc complained that the spsc queues reference it but it is not
> visible enough.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/ReactiveX/RxJava/pull/4039/files/12e00d19d0ac9c82eefd8f83a6f6bba7e0ea53b2#r67684701>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA7B3HF0gIVikH3WAb6RfWHi7_UVir0Lks5qNo78gaJpZM4I5a4v>
> .
>